test/e2e/atlas/data/connected_org_config.json

```diff
@@ -0,0 +1,13 @@
+{
+  "domainAllowList": ["https://accounts.google.com"],
+  "domainRestrictionEnabled": false,
+  "userConflicts": [
+    {
+    "emailAddress": "user@example.com",
+    "federationSettingsId": "656e4d8e91cb7d26db1bc9c6",
```

Comment:
is this a real ID? and if yes? can you move this to be a config on evg?

Reply:
yes but it can be an invalid ID, it was an accident, updated